src/main/java/com/wolox/training/models/Book.java

```diff
@@ -56,6 +58,9 @@
     @NotNull
     private String isbn;
 
+    @ManyToMany(mappedBy = "books")
+    private List<User> users;
```

Comment:
add `@JsonIgnore` to avoid infinite recursion

Reply:
Solved, i add @JsonIgnore